index.js

```diff
@@ -88,6 +89,11 @@ Object.defineProperties(MediaSession.prototype, {
             }
             return [];
         }
+    },
+    channels: {
+        get: function () {
+            return this.pc._remoteDataChannels.concat(this.pc._localDataChannels);
```

Comment:
oddly, there is no equivalent to getLocalStreams/getRemoteStreams. Managing the datachannel lifecycle is app reponsibility. I don't think the media session is the right place even though I agree we should be emitting channelAdded to enable the application (or any modules on top of this) to handle this.

tl;dr: can we just do onAddChannel? ;-)